vhost/src/vhost_user/gpu_backend_req.rs

```diff
@@ -164,6 +165,12 @@ impl GpuBackend {
             .map_err(|e| io::Error::new(io::ErrorKind::Other, format!("{}", e)))
     }
 
+    /// Forward 2d gpu command Update Scanout requests to the frontend.
+    pub fn get_update(&mut self, update: &VhostUserGpuUpdate) -> io::Result<()> {
```

Comment:
The name `get_update` seem a misleading to me - It sounds like a getter but it not really a getter (doesn't even return any value other than a potential error).
If i wanted to stick with the original name I guess it would be just `update` but maybe `update_scanout` could be more informative / less weird for a method name. What do you think?